test/performance/dataplane-probe/main.go

```diff
@@ -74,12 +75,28 @@ func main() {
 		fatalf("Unrecognized target: %s", *target)
 	}
 
+	// Make sure the target is ready before sending the large amount of requests.
+	spoofingClient := spoof.SpoofingClient{
+		Client:          &http.Client{},
+		RequestInterval: 1 * time.Second,
+		RequestTimeout:  *duration,
+		Logf: func(fmt string, args ...interface{}) {
+			log.Printf(fmt, args)
```

Comment:
Can you just assign `log.Printf` here? Do you need to wrap it in another function?